src/Compilers/CSharp/Portable/Symbols/DeclarationScope.cs

```diff
@@ -2,16 +2,28 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Roslyn.Utilities;
+
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
-    // https://github.com/dotnet/roslyn/issues/61647: Internally, scope is represented with this enum,
-    // but the public API uses a pair of IsRefScoped and IsValueScoped bools (see ILocalSymbol,
-    // IParameterSymbol, and ScopedRefAttribute). We should have a common representation.
-    // And we should use common terms for the attribute and enum names.
     internal enum DeclarationScope : byte
```

Comment:
>DeclarationScope [](http://example.com/codeflow?start=18&length=16)

Replace with `ScopedKind`?

Reply:
Seems like it would be a reasonable change.

Reply:
https://github.com/dotnet/roslyn/issues/64248